WcaOnRails/app/models/competition.rb

```diff
@@ -357,6 +357,10 @@ def warnings_for(user)
         warnings[:name] = I18n.t('competitions.messages.name_too_long')
       end
 
+      if (self.registration_open - Time.now) > 172_800
```

Comment:
Would make sense to move this value to a constant like [this](https://github.com/thewca/worldcubeassociation.org/blob/16dca929c434e83f15a4d3b7e92cfbca73b026b1/WcaOnRails/app/models/competition.rb#L212).

Reply:
It probably would, yes.